%ProgramData%/Microsoft/Windows/UUS/State/_active.uusver

```diff
@@ -1 +0,0 @@
-1308.2407.15042.0
```

Comment:
Probably best to stick an entry in .gitignore to make sure these don't get in again e.g.

%ProgramData%/

Reply:
Sure, should I do that as a commit on this or another branch?